### DISCUSSION TITLE
cannot use sticky inside a frame widget, it always sticks to west, i cannot move it

### DISCUSSION THREAD

**Discussion by nitinss17** [TomSchimansky/CustomTkinter#1954] [General]
Hello, 

if i set 'ne' it should stick to east but its sticking to west, this is observed only when widgets are inside a frame, if i keep it outside its working as intended, please help! 

code : to copy paste and test 

import customtkinter

class App(customtkinter.CTk):
    def __init__(self):
        super().__init__()

        self.grid_columnconfigure(0, weight=1)
        self.grid_rowconfigure(0, weight=1)

        self.top_frame_no_inspection = customtkinter.CTkFrame(self)
        self.top_frame_no_inspection.grid(row=0, column=0, padx=(10,10), pady=(10,10),sticky="new")

        self.button_start_inspection = customtkinter.CTkButton(self.top_frame_no_inspection, text="Start")
        self.button_start_inspection.grid(row=0, column=0, padx=(10,10), pady=(10,10), sticky="ne") #not working 

if __name__ == "__main__":
    customtkinter.set_appearance_mode("dark")  
    customtkinter.set_default_color_theme("green")  
    app = App()
    app.geometry("640x480")
    app.mainloop()

<img width="488" alt="sticky" src="https://github.com/TomSchimansky/CustomTkinter/assets/58427646/83ced955-77a8-42fe-8e67-90af5ed2f4f0">

**Comment by Akascape** [comment]
@nitinss17 
set grid configure method for the frames too.

**Reply by nitinss17** [reply]
Thank you. this helped to solve it.
